compiler/src/dotty/tools/dotc/ast/untpd.scala

```diff
@@ -736,6 +737,8 @@ object untpd extends Trees.Instance[Untyped] with UntypedTreeInfo {
         x
       case TypedSplice(splice) =>
         this(x, splice)
+      case MacroTree(call) =>
```

Comment:
I believe it's more systematic and safer to also add MacroTree clauses for copying and mapping trees. When adding a new tree node, I usually search for all occurrences of another similar node and then do alike for the new node.

Reply:
Added